src/main/java/nl/tudelft/dnainator/ui/widgets/ProgressDialog.java

```diff
@@ -33,8 +36,11 @@ private void setupAlert() {
 		alert.setTitle(" ");
 		alert.setHeaderText("Loading...");
 		alert.getButtonTypes().add(ButtonType.CANCEL);
-
+		
 		alert.getDialogPane().setContent(progressBar);
+		Stage stage = (Stage) alert.getDialogPane().getScene().getWindow();
```

Comment:
What is this for? If you want to set an icon for the Stage, please do so in the `DNAInator` class.